homeworks/03-data-structure/src/SimpleBinaryTree.ts

```diff
@@ -0,0 +1,83 @@
+import BinaryTree, { TreeNode, TraverseType } from './BinaryTree'
+
+export default class SimpleBinaryTree<T> implements BinaryTree<T> {
+
+  constructor(protected root: TreeNode<T>) {
+  }
+
+  getRoot(): TreeNode<T> {
+    return this.root
+  }
+
+  setRoot(root: TreeNode<T>): this {
```

Comment:
Always define access modifiers.

Reply:
public in this case? It's default option, what's the point to repeat defaults?

Reply:
Just to be consistent and write to a readable and understandable code base. The code becomes much more obvious and it looks in a common typed language style.